### ISSUE TITLE
Can't go install due to replace directive

### ISSUE THREAD

**Issue by znd4** [quexten/goldwarden#219]
Hi, sorry I don't know of a workaround for this, but I'm getting the following error with `go install ...@latest`:

```
go: github.com/quexten/goldwarden@latest (in github.com/quexten/goldwarden@v0.3.2):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

see [this go issue](https://github.com/golang/go/issues/44840)

FWIW, the latest installable semver tag seems to be `v0.2.15`

**Comment by quexten** [comment]
Had the rename because I submitted a bugfix PR to an upstream dependency, and the replace changed it to my fork. Since upstream merged my PR and released a new version, #220 should fix this.

**Comment by quexten** [comment]
Released 0.3.3 containing this fix.
